yield_plugin/src/yield_plugin.cpp

```diff
@@ -707,7 +717,7 @@ namespace yield_plugin
   double YieldPlugin::polynomial_calc_d(std::vector<double> coeff, double x) const
   {
     double result = 0;
-    for (size_t i = 0; i < coeff.size()-1; i++) 
+    for (size_t i = 0; i < coeff.size()-1; i++)
     {
       double value = (int)(coeff.size() - 1 - i) * coeff.at(i) * pow(x, (int)(coeff.size() - 2 - i));
```

Comment:
These casts should be changed from C-style casts to `static_cast<int>()`.

Reply:
added to int and doubles